README.md

```diff
@@ -1,4 +1,26 @@
-# signal-desktop
+<h1 align="center">
+  <img src="https://signal.org/assets/header/logo-f7ef605fe417d5520d38d546b3b774b4261c75220b9904da4d8b2ffc19a761ff.png" alt="Signal Desktop">
+</h1>
 
-Previously signal was built from source.
-Now we simply ingest the latest deb directly from upstream and bundle it in a snap
+<p align="center"><b>This is the snap for Signal Desktop</b>. It is a community-maintained package to easily install Signal Desktop on Ubuntu, Fedora, Debian and other major Linux distributions. It is available in the Snap Store, Ubuntu Software, and a number of other applications.</p>
+
+<p align="center"><i>"Signal Desktop is a private messaging app for your desktop"</p>
+
+<p align="center">
+<a href="https://snapcraft.io/signal-desktop"><img src="https://snapcraft.io/signal-desktop/badge.svg" alt="Snap Status"></a>
+</p>
+
+## Install
+
+```shell
+snap install signal-desktop
+```
+
+([Don't have snapd installed?](https://snapcraft.io/docs/core/install))
+
+<p align="center">Published for <img src="https://raw.githubusercontent.com/anythingcodes/slack-emoji-for-techies/gh-pages/emoji/tux.png" align="top" width="24" /> with :gift_heart: by Snapcrafters</p>
+
+## License
+
+* The license of the build files in this repository is GNU Affero General Public License v3.0 only
+* Signal Desktop itself has [its own license](https://github.com/signalapp/Signal-Desktop/blob/main/LICENSE).
```

Comment:
It looks like Signal Desktop uses the GNU AGPL 3.0 only, too so you can get away with dropping this line, maybe?

Reply:
Agreed, I'll drop it in a following PR.